.github/workflows/auto-release.yml

```
@@ -0,0 +1,282 @@
+        pip install -r requirements-dev.txt
+
+    - name: Determine version bump type
+      id: version-bump
+      run: |
+        # Check if this is a manual dispatch
+        if [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
+          echo "bump_type=${{ github.event.inputs.version_bump }}" >> $GITHUB_OUTPUT
+          echo "Manual dispatch: Using ${{ github.event.inputs.version_bump }} bump"
+          exit 0
+        fi
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)